packages/apollos-server-core/src/giving/simple-donation.js

```diff
@@ -0,0 +1,20 @@
+export default function useSimpleDonationRoute({ app }) {
+  // Give Screen
+  app.get('/simpledonation', (req, res) => {
+    res.send(`
+            <!DOCTYPE html>
+            <html>
+                <head>
+                    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+                </head>
+                <body>
+                    <script async src="https://merlin.simpledonation.com/js/installScript.js?key=${process.env.MERLIN_KEY}&cssFile=https://tom-misc.s3.us-east-2.amazonaws.com/giving.css"></script>
```

Comment:
should this be?
```suggestion
                    <script async src="https://merlin.simpledonation.com/js/installScript.js?key=${ApollosConfig.MERLIN.KEY}&cssFile=https://tom-misc.s3.us-east-2.amazonaws.com/giving.css"></script>
```

We have moved away from storing configuration in the env.

Reply:
Yep I can do that. Is there a certain way to add it to the ApollosConfig? I think that's why I didn't do it because I couldn't figure it out

Reply:
Yah :D You can add something like this to the end of the `config.yml` in templates

```
MERLIN: 
  KEY: ""
```

Reply:
Nvm I figured it out